github_copilot_seats/triggers.py

```diff
@@ -0,0 +1,28 @@
+from autokitteh.slack import slack_client
+from seats import prune_idle_seats, find_idle_seats
+
+s = slack_client("slack_conn")
+
+
+def on_schedule() -> None:
+    print(prune_idle_seats())
+
+
+def on_slack_slash_command(event) -> None:
+    cmd = event.data.text
```

Comment:
Cheap normalization for extra safety:
```suggestion
    cmd = event.data.text.lower()
```